test/1.4/misc/dead_methods_imported.dml

```diff
@@ -0,0 +1,48 @@
+/*
+  © 2023 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+/// DEAD
+method dead_imported() -> (int) {
+    return 0;
+}
+
+template t {
+    // no problem
+    shared method dead_abstract();
+    // TODO: this one is apparently dead, but C code is still generated
```

Comment:
You have this dangling TODO. Has it been addressed?

Reply:
No, I did not find it important. Ideally we may want to make dead method elimination more complete, but I fear it may be a rabbit hole I don't want to get stuck in today.

Dead shared methods isn't a big problem, and I suppose they will be reported by coverage analysis.

Reply:
Alright.